src/models/booking.cr

```diff
@@ -51,6 +94,9 @@ class Booking
   column created : Int64?
 
   column extension_data : JSON::Any, presence: false
+  column history : Array(History), presence: false
+
+  property utm_source : String? = nil
```

Comment:
Is this the correct way of adding a property to a Clear model that does not need to be stored in the database?

Reply:
Why isn't this being stored in the DB?

Reply:
the utm_source is included in the history, so I didn't think there would be any value to storing it separately as well.

Reply:
Ok cool